src/macros.rs

```diff
@@ -56,13 +58,15 @@ macro_rules! redis_command {
             return $crate::raw::Status::Err as c_int;
         }
 
+        let mandatory_acl_categories = AclCategory::from($mandatory_acl_categories);
         if let Some(RM_SetCommandACLCategories) = $crate::raw::RedisModule_SetCommandACLCategories {
-            let mut acl_categories = CString::new("").unwrap();
+            let mut acl_categories = CString::default();
             $(
-                if $mandatory_acl_categories != "" && $optional_acl_categories != "" {
-                    acl_categories = CString::new(format!("{} {}", $mandatory_acl_categories, $optional_acl_categories)).unwrap();
-                } else if $optional_acl_categories != "" {
-                    acl_categories = CString::new($optional_acl_categories).unwrap();
+                let optional_acl_categories = AclCategory::from($optional_acl_categories);
+                if mandatory_acl_categories != AclCategory::None && optional_acl_categories != AclCategory::None {
+                    acl_categories = CString::new(format!("{} {}", mandatory_acl_categories, optional_acl_categories)).unwrap();
+                } else if optional_acl_categories != AclCategory::None {
+                    acl_categories = CString::new(format!("{}", $optional_acl_categories)).unwrap();
```

Comment:
Can we just `$optional_acl_categories.to_string()`?

Reply:
it needs to be a `c_string` to pass to `Redis`

Reply:
I know, I meant instead of using the `format!` macro we could have simply used the `to_string()` method of the `fmt::Display` trait, no?